<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="MView" name="view" native="true"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,1,8,8">
        <item>
         <widget class="QPushButton" name="btnOpenDir">
          <property name="text">
           <string>Open Dir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnChangeSaveDir">
          <property name="text">
           <string>Change Save Dir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNextImage">
          <property name="text">
           <string>Next Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPrevImage">
          <property name="text">
           <string>Prev Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>File Url</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView"/>
        </item>
        <item>
         <widget class="DisplayWidget" name="displayWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开页面</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MView</class>
   <extends>QWidget</extends>
   <header>mview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DisplayWidget</class>
   <extends>QWidget</extends>
   <header location="global">displaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
